microsoft/testsuites/performance/networkperf.py

```diff
@@ -108,14 +115,72 @@ def perf_tcp_max_pps_synthetic(self, environment: Environment) -> None:
         ),
     )
     def perf_tcp_max_pps_sriov(self, environment: Environment) -> None:
-        self.perf_tcp_pps(environment, "perf_tcp_max_pps_sriov", "maxpps")
+        self.perf_tcp_pps(environment, "maxpps")
+
+    @TestCaseMetadata(
+        description="""
+        This test case uses ntttcp to test synthetic tcp network throughput.
+        """,
+        priority=3,
+        timeout=12000,
+        requirement=simple_requirement(
+            min_count=2,
+            network_interface=Synthetic(),
+        ),
+    )
+    def perf_tcp_ntttcp_synthetic(self, log: Logger, environment: Environment) -> None:
```

Comment:
Let's add one P2 test case to test only 1 typical tcp connections like 128? It's fast and can be test on more scenarios.

Reply:
add perf_tcp_ntttcp_128_connections_synthetic